.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: MeshKit
+on:
+  push:
+    branches: [ master ]
```

Comment:
In order to detect problems and fail early this should also run on pushes to all branches, and I think also tags:

```on:
  push:
    branches:
      - '**'
    tags:
      - 'v*'
  pull_request:
    branches:
      - master
```